astrometry/include/PixelMap.h

```diff
@@ -332,5 +334,37 @@ namespace astrometry {
     void operator=(const ColorTerm& rhs) = delete;
   
   };
+
+  class ASTMap: public PixelMap {
+    // Class representing a constant shift of position
+  public:
+    ASTMap(const ast::Mapping& mapping_, string name_=""):
+      PixelMap(name_), mapping(mapping_) {}
+
+    virtual PixelMap* duplicate() const;
+
+    ~ASTMap() {}
```

Comment:
can be declared as default dtor
```suggestion
    ~ASTMap() = default;
```